set-definitions/src/main/java/eu/europeana/set/definitions/model/impl/BaseUserSet.java

```diff
@@ -371,4 +215,34 @@ public boolean isPublic() {
     public boolean isPublished() {
 	return VisibilityTypes.PUBLISHED.getJsonValue().equals(getVisibility());
     }
+
+    /**
+     * if List<String>subject contains an entity reference
+     * referring to the set then it is a EntityBestItemSet
+     *
+     * @return true if it is an EntityBestItemSet
+     */
+    public boolean isEntityBestItemSet() {
```

Comment:
this method should simply verify the type of the set to be EntityBestItemSet

Reply:
there is a method isEntityBestItemSet in the service with the correct implementation. I think we can remove this one.

Reply:
Bt this method is to verify if the subject filed is present and has entity reference. Ther service implementation one just checks the type of the set. Either i can add this logic there or we keep this method. This something like isOpenSet

Reply:
have moved the business logic to service class

Reply:
the isOpenSet is somethinf different, becasue it has the same type as the closed set. But entity sets have their own type